### PR TITLE
Add reactive extension to Kingfisher ImageCache retriveImageInDiskCache

### PR DESCRIPTION
**Pull request by aksswami**
Adding ImageCache reactive extension. 
ImageCache provide completion callback for retrieving from cache (disk or memory), wanted to use this in Rx codebase, so added extension for the same.

### CODE REVIEW

**Comment by aksswami** [comment]
@freak4pc any comments, issues, review for the PR?

File: Sources/ImageCache+Rx.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+//  Created by Amit Kumar Swami on 4/3/19.
+//  Copyright © 2019 RxSwift Community. All rights reserved.
+//
+
+import RxSwift
+import Kingfisher
+
+extension Reactive where Base: ImageCache {
+    public func retrieveImageInDiskCache(
+        forKey key: String,
```
Comment (freak4pc) [inline Sources/ImageCache+Rx.swift:14]: Could you move the first argument to the same line as the method signature?

File: Sources/ImageCache+Rx.swift [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        options: KingfisherOptionsInfo? = nil) -> Single<Image?> {
+        return Single<Image?>.create { [base] single in
+            base.retrieveImageInDiskCache(forKey: key, options: options) { result in
+                switch result {
+                case let .failure(error):
```
Comment (freak4pc) [inline Sources/ImageCache+Rx.swift:19]: Could you change this and the success case to have the `let` on the argument? I personally use a `case let` only if there's more than one argument.

```suggestion
                case .failure(let error):
```

**Review by freak4pc — CHANGES_REQUESTED** [review]
Looks great! Just two minor nitpicks to align the formatting to other extensions in this repo. 

Also, please add a proper "Added" CHANGELOG entry. 

Thanks!

**Pull request closed without merging**